app/decorators/data_request_decorator.rb

```diff
@@ -17,10 +17,6 @@ def request_dates
     ""
   end
 
-  def location
-    contact&.name || super
-  end
-
   def data_request_name
     contact&.data_request_name || location
```

Comment:
Does this need changing to remove location?

Reply:
My thinking was that the location is now an attribute of data_request_area so this wasnt needed